distributed/system_monitor.py

```diff
@@ -38,14 +40,22 @@ class SystemMonitor:
     # distributed.admin.system_monitor.interval = 500ms
     def __init__(
         self,
-        maxlen: int | None = 7200,
+        maxlen: int | None | NoDefault = no_default,
         monitor_disk_io: bool | None = None,
         monitor_host_cpu: bool | None = None,
         monitor_gil_contention: bool | None = None,
     ):
         self.proc = psutil.Process()
         self.count = 0
+
+        if maxlen is no_default:
+            maxlen = dask.config.get("distributed.admin.log-length")
+        if isinstance(maxlen, int):
+            maxlen = max(1, maxlen)
```

Comment:
I'm not sure how I feel about lumping this in with the other logs.

Barely anybody will care for the scheduler/worker transition log

However, this will cut the system dashboard page, won't it?

Reply:
Moved to `distributed.admin.system-monitor.log-length`